client/public/index.html.ejs

```diff
@@ -2,14 +2,18 @@
 <html lang="en-US">
   <head>
     <meta charset="utf-8" />
-    <title>Tackle</title>
-    <link rel="icon" href="favicon.ico" />
+    <% if(brandType == 'mta'){ %>
+      <title>Migration Toolkit for Applications</title>
+      <link rel="icon" href="mta-favicon.ico" />
+      <% } else{ %>
+      <title>Tackle</title>
+      <link rel="icon" href="konveyor-favicon.ico" />
+      <% } %>
     <meta id="appName" name="application-name" content="Konveyor Tackle UI" />
```

Comment:
We probably want to have these `<meta>` tags for `application-name` and `description` also use the right brand name I think? I'm not sure how those get used.